content/en/user-manual/publishing/mobile/cordova.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Apache Cordova
+template: usermanual-page.tmpl.html
+position: 1
+---
+
+[Apache Cordova][1] is an open-source mobile development framework. It allows you to use standard web technologies - HTML5, CSS3, and JavaScript for cross-platform development. Applications execute within wrappers targeted to each platform, and rely on standards-compliant API bindings to access each device's capabilities such as sensors, data, network status, etc.
+
+You can use Cordova to natively wrap your PlayCanvas app. You can then publish it to the iOS App Store and Android's Google Play. Cordova can also generate executables compatible with macOS and Windows.
```

Comment:
'You can' twice in a row for the start of sentences sounds odd.

Maybe:
You can use Cordova to natively wrap your PlayCanvas app. This can be published to the iOS App Store and Android's Google Play.